bower.json

```diff
@@ -8,6 +8,7 @@
     "json3": "3.3.1",
     "es5-shim": "3.1.0",
     "jquery": "2.1.1",
+    "jquery-visibility": "1.0.11",
```

Comment:
@kaizo I can't see where we are loading this file (Maybe you forgot to require it)